### PR TITLE
Added jupyter book links and corrected path for figures

### LINKED ISSUE #1092: Broken links in README.md inside the /book directory
## Bug description
- [ ] The link under the subsection **Configuration** under *README.md* file inside `/book` is not turning up
- [ ] Incorrect path for searching out figures written on the top as `website/assets/figures/`

## Expected behaviour
- [ ] The link should be redirected to correct web page
- [ ] The path for figures should be corrected

##  personal set up
- OS:
- [x] Linux

- Browser:
- [x] Chrome
- [x] Firefox

### PR DESCRIPTION
**Pull request by IIITM-Jay**
## Summary
Fixes #1092 

Done:
- [x] The link should be redirected to correct webpage
- [x] Corrected path for searching out the figures

### CODE REVIEW

File: book/README.md [hunk 1]
```diff
@@ -3,11 +3,11 @@
 
 All the text for each chapter lives inside its own folder in the `content/` directory.
 
-Everything else is in the `website/`. Importantly this includes the figures, which are in the `website/assets/figures/` directory.
+Everything else is in the `website/`. The figures are present in the `book/content/figures/` directory.
```
Comment (malvikasharan) [inline book/README.md:6]:
```suggestion
All figures associated to the chapters are stored in and linked from the `book/content/figures/` directory.
Everything else is in the `website/` folder.
```

File: book/README.md [hunk 2]
```diff
@@ -3,11 +3,11 @@
...
 
 ### Configuration
-- To change the table of contents (order of the chapters) see the `website/_data/toc.yml` file. Documentation can be found on the [jupyter book website](https://jupyter.org/jupyter-book/intro.html).
-- Same applies for more general configuration using `website/_config.yml`
+- To change the table of contents (order of the chapters) see the `website/_data/toc.yml` file. Documentation can be found on the [TOC structure](https://jupyterbook.org/customize/toc.html).
```
Comment (malvikasharan) [inline book/README.md:9]:
```suggestion
- The table of contents (TOC) defines the order of the chapters as they appear in the book.
To change the TOC, please edit the `website/_data/toc.yml` file with correct information on the location of chapters, their corresponding file names and the relative location of the files. 
Documentation on controlling the TOC structure can be found on the [jupyter book website](https://jupyterbook.org/customize/toc.html).
```

File: book/README.md [hunk 3]
```diff
@@ -3,11 +3,11 @@
...
+- Same applies for more general configuration using `website/_config.yml` whose documentation can be viewed on [Configure book settings](https://jupyterbook.org/customize/config.html)
```
Comment (malvikasharan) [inline book/README.md:10]:
```suggestion
- Same applies for more general configuration using `website/_config.yml`. 
Documentation on configuring book settings can be found on the [jupyter book website](https://jupyterbook.org/customize/config.html).
```

**Comment by IIITM-Jay** [comment]
Thanks @malvikasharan . I will surely follow up with the suggestions.

**Comment by malvikasharan** [comment]
@IIITM-Jay You can accept commits directly in this PR, or edit them as a comment.

File: book/README.md [hunk 4]
```diff
@@ -7,7 +7,9 @@ All figures associated to the chapters are stored in and linked from the `book/c
 Everything else is in the `website/` folder.
 
 ### Configuration
-- To change the table of contents (order of the chapters) see the `website/_data/toc.yml` file. Documentation can be found on the [TOC structure](https://jupyterbook.org/customize/toc.html).
+- The table of contents (TOC) defines the order of the chapters as they appear in the book.
+To change the TOC, please edit the `website/_data/toc.yml` file with correct information on the location of chapters, their corresponding file names and the relative location of the files. 
+Documentation on controlling the TOC structure can be found on the [jupyter book website](https://jupyterbook.org/customize/toc.html).
```
Comment (malvikasharan) [inline book/README.md:12]:
```suggestion
- The table of contents (TOC) defines the order of chapters as they appear in the book.
To change the TOC, please edit the `website/_data/toc.yml` file with correct information on file names and their relative locations in this repository. 
Documentation on controlling the TOC structure can be found on the [jupyter book website](https://jupyterbook.org/customize/toc.html).
```

**Comment by malvikasharan** [comment]
Thanks @IIITM-Jay. I will merge this PR.

**Pull request merged**